.github/instructions/sync.instructions.md

```
@@ -0,0 +1,72 @@
+---
+applyTo: "**/views/base/**,**/views/object_sync/**,**/views/sync/**,**/tables/**,**/librenms_sync.js"
+description: Sync page architecture, base views, and sync action patterns
+---
+
+# Sync Pages
+
+## Three-Layer View Architecture
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)